doc/decisions/DR014-Management-policy-responses.md

```diff
@@ -0,0 +1,307 @@
+critical part of the OpenAssetIO abstraction.
+
+OpenAssetIO classifies entities (assets) using a Trait Set, which lists
+one or more traits that describe its nature. For example, an image
+file may have the trait set `{'file', 'image', 'colorManaged'}`.
+
+The `managementPolicy` method is a high-level (i.e. not entity specific)
+query that a host can use to determine a manager's behaviour in relation
+to a specific trait set.
+
+The answer to this query is commonly used by a Host to conditionally
+enable user-facing functionality based on the behaviour of the specific
+manager that is in use.
+
+This mechanism is flexible, and so exactly how a manager should
+implement its response is potentially ambiguous. This document outlines
+the rationale behind the defined canonical behaviour.
+
+## Relevant data
+
+Documentation pertaining to the OpenAssetIO data model:
+
+- [Entities, Traits and Specifications overview](../doxygen/src/EntitiesTraitsSpecifications.dox)
+- [Compositional data model decision record](./DR007-Hierarchical-or-compositional-traits-for-specifications.md)
+- [Unified Entity data model decision record](./DR008-Unify-the-entity-data-model.md)
+
+Notable points:
+
+- A Trait is how OpenAssetIO describes a behaviour or quality.
+- Traits consist of a unique ID, and zero or more simple-typed
+  properties that further describe that trait.
+- A Trait Set is an un-ordered collection of unique trait IDs.
+- Trait Sets combine multiple traits to increase specificity, rather
+  than meaning "or".
+- A `TraitsData` instance is a dict of dicts, that can be "imbued" with
+  zero or more traits and the values for any associated properties those
+  traits may have. The top-level keys of this container form a Trait
+  Set.
+- The `managementPolicy` result is a `TraitsData` instance for each
+  supplied Trait Set.
+- Some managers may only care about certain specific Traits (such as
+  those describing the assets content disposition), and handle all
+  'derivative entity types' the same.
+- The `managementPolicy` query is not a required pre-requisite in the
+  use of `resolve`, et. al. but is used to adapt application behaviour
+  to improve user experience.
+
+## Illustrative scenario
+
+Consider a manager that manages the files for assets used in 2D
+compositing.
+
+It happily manages the node graphs used to define a comp, plus any
+associated images read/written by the process. It does however, have
+no interest in managing intermediate cache files that may be used
+to speed up interactivity.
+
+Its implementation pivots all behaviour around two (imaginary) traits:
+
+- The `file` trait - indicating the entity's data is stored in a file.
+- The `cache` trait - indicating that the data is re-creatable.
+
+The manager has a fairly simple database and isn't capable of storing
+arbitrary data for an entity, just the appropriate file path. It
+fulfills the API contract, by ensuring it can also store an entity's
```

Comment:
I've removed that section to avoid clouding the issue as part of the re-write [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/792/commits/e813a8bc4bd9c43a1eba53c446cce51e446f9608). See if that helps?

Reply:
Yeah its much clearer now, though we should make it super explicit in future API docs.